src/routes/open/container/Open.tsx

```diff
@@ -24,21 +24,52 @@ import { userAccountSelector } from 'src/logic/wallets/store/selectors'
 import { SafeRecordProps } from 'src/logic/safe/store/models/safe'
 import { addOrUpdateSafe } from 'src/logic/safe/store/actions/addOrUpdateSafe'
 import { PromiEvent, TransactionReceipt } from 'web3-core'
+import { useLocation } from 'react-router-dom'
 
 const SAFE_PENDING_CREATION_STORAGE_KEY = 'SAFE_PENDING_CREATION_STORAGE_KEY'
 
-const validateQueryParams = (ownerAddresses, ownerNames, threshold, safeName) => {
+interface SafeCreationQueryParams {
+  ownerAddresses: string | string[] | null
+  ownerNames: string | string[] | null
+  threshold: string | string[] | null
+  safeName: string | string[] | null
+}
+
+export interface SafeProps {
+  name: string
+  ownerAddresses: string[]
+  ownerNames: string[]
+  threshold: string
+}
+
+const validateQueryParams = (queryParams: SafeCreationQueryParams): boolean => {
+  const { ownerAddresses, ownerNames, threshold, safeName } = queryParams
   if (!ownerAddresses || !ownerNames || !threshold || !safeName) {
     return false
   }
   if (!ownerAddresses.length || ownerNames.length === 0) {
     return false
   }
 
-  if (Number.isNaN(Number(threshold))) {
+  if (!threshold) {
```

Comment:
is Think that is it ok to validate if the value is a number, what would happen if `?threshold=DiegoMaradona`?

Reply:
It will be converted to 10 😜. Good catch, will add it

Reply:
:laughing: